tests/samples_smoke_tests/common/common_utils.py

```diff
@@ -0,0 +1,76 @@
+"""
+ Copyright (C) 2018-2021 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+import glob
+import os
+import re
+import subprocess
+import sys
+import numpy as np
+
+def shell(cmd, env=None, cwd=None, out_format="plain"):
+    """
+    Run command execution in specified environment
+    :param cmd: list containing command and its parameters
+    :param env: set of environment variables to set for this command
+    :param cwd: working directory from which execute call
+    :param out_format: 'plain' or 'html'. If 'html' all '\n; symbols are replaced by '<br>' tag
+    :return:
+    """
+    if sys.platform.startswith('linux') or sys.platform == 'darwin':
+        cmd = ['/bin/bash', '-c', "unset OMP_NUM_THREADS; " + " ".join(cmd)]
```

Comment:
`unset OMP_NUM_THREADS;` is legacy and may be removed

Reply:
Was removed.